pco_python/src/lib.rs

```diff
@@ -92,12 +166,21 @@ impl PyChunkConfig {
   /// :param delta_encoding_order: either a delta encoding level from 0-7 or
   /// None. If set to None, pcodec will try to infer the optimal delta encoding
   /// order.
-  /// :param int_mult_spec: either 'enabled' or 'disabled'. If enabled, pcodec
-  /// will consider using int mult mode, which can substantially improve
-  /// compression ratio but decrease speed in some cases for integer types.
-  /// :param float_mult_spec: either 'enabled' or 'disabled'. If enabled, pcodec
-  /// will consider using float mult mode, which can substantially improve
-  /// compression ratio but decrease speed in some cases for float types.
+  /// :param int_mult_spec: a IntMultSpec disabling, enabling, or providing a
```

Comment:
I don't think we should list the available options here - better for people to look at the `*Spec` docs for that.

Reply:
Changed